new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,297 @@
+# DynamoDB Integration with Typesense
+
+Hey there! This post give you a guide to integrate Typesense custer with AWS DynamoDB by setting up a trigger with AWS Lambda.
+
+![Typesense DynamoDB Integration Chart](~@guide-images/typesense-dynamodb/dynamodb-typesense-integration-1.png)
+
+## Create AWS DynamboDB with streams enabled
```

Comment:
Could you add Step numbers and make the titles shorter in general. 

See this article for an example: https://new-site.typesense.org/docs/0.19.0/guide/typesense-firebase.html

For eg: this particular title could be shortened to "Step 1: Create a DynamoDB table"